requirements.txt

```diff
@@ -4,18 +4,17 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-amqp==1.4.9               # via kombu
-ansible==2.1              # via subspace
-anyjson==0.3.3            # via kombu
+-e git+https://github.com/steve-gregory/billiard.git#egg=billiard
+amqp==2.1.4               # via kombu
+ansible==2.2.1.0          # via subspace
```

Comment:
Yes, subspace `0.3.0` requires ansible `2.2` due to changes in the `task_queue_manager`, and `subspace` strategy-plugin. (The price we pay for private method usage!)